plugin/storage/es/dependencystore/storage.go

```diff
@@ -31,36 +31,50 @@ import (
 )
 
 const (
-	dependencyType  = "dependencies"
-	dependencyIndex = "jaeger-dependencies-"
+	dependencyType       = "dependencies"
+	dependencyIndex      = "jaeger-dependencies-"
+	indexPrefixSeparator = "-"
 )
 
 // DependencyStore handles all queries and insertions to ElasticSearch dependencies
 type DependencyStore struct {
-	ctx         context.Context
-	client      es.Client
-	logger      *zap.Logger
-	indexPrefix string
+	ctx                   context.Context
+	client                es.Client
+	logger                *zap.Logger
+	dependencyIndexPrefix string
+	useReadWriteAliases   bool
+}
+
+// DSParams holds constructor parameters for NewDependencyStore
+type DSParams struct {
```

Comment:
Can we use the full name `DependencyStoreParams` to be consistent with the span store?

Reply:
Certainly. I had that exact name before, but the linter was complaining ;-)